app/controllers/api/v1/upload_controller.rb

```diff
@@ -11,10 +11,14 @@ def create
       response = s3.put_object(bucket: ENV['S3_BUCKET'], key: key, body: params[:audio])
 
       if response.etag
+        # verse = Verse.create(
+        #   audio_path: key,
+        #   title: params[:title],
+        #   user_id: params[:userId],
+        #   track_id: params[:trackId]
+        # )
         verse = Verse.create(
-          audio_path: key,
-          user_id: params[:userId],
-          track_id: params[:trackId]
+          verse_params.merge({audio_path: key})
```

Comment:
This is a pretty easy refactor that should be useful in the future; definitely want to remember to go back over this and change to use this syntax anywhere we're creating a new resource in this controller